portal-web/test/functional/com/liferay/portalweb/macros/Clustering.macro

```diff
@@ -272,6 +272,25 @@ definition {
 		}
 	}
 
+	macro waitForNodeDeactivation {
+		var nodeLog = Clustering.getLogFileSpecificServer(nodePort = ${nodePort});
+
+		var minutestoExpire = RegexUtil.replace(${nodeLog}, "temporarily permitted during the grace period which expires in (.*[0-9]) minutes", 1);
+
+		var mstoExpire = ${minutestoExpire} * 60000;
+
+		var mstoExpire = ${mstoExpire} + 60000;
```

Comment:
Itll be more readable if you add the minute leeway before the conversion from minutes to ms and add it there with a comment for context, since its hard for most people to be able to associate 60000ms to 1 minute